coss/home/static/js/clubs_about.js

```diff
@@ -0,0 +1,29 @@
+$(document).ready(function() {
```

Comment:
Why is this moved in the `home` app?

Reply:
All the javascript code there is needed for the parts of the template that live in the home app. I'll rename it to a more generic name.